rvs/CMakeLists.txt

```diff
@@ -166,6 +166,22 @@ install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/.rvsmodules.config"
   COMPONENT applications
 )
 
+option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "Build with file/folder reorg with backward compatibility enabled" ON)
+if(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY)
+  #Create the softlink rvs/rvs->../bin/rvs in install path for backward compatibility
```

Comment:
Symbolic link created just for backward compatibility purpose. But not used by any scripts for now right ?

Reply:
I dont think any internal scripts use this symlink.